cartoframes/data/observatory/catalog/entity.py

```diff
@@ -118,9 +118,20 @@ def _get_print_id(self):
 
         return self.id
 
-    def _download(self, credentials, file_path=None, limit=None, order_by=None):
+    def _download(self, credentials, file_path=None, limit=None, order_by=None, sql_query=None, add_geom=None):
         auth_client = credentials.get_api_key_auth_client()
-        rows = DODataset(auth_client=auth_client).name(self.id).download_stream(limit=limit, order_by=order_by)
+
+        is_geography = None
```

Comment:
It's an internal param when using the `Geography` class is set to `True`. We need to detect when the user wants to download the geography dataset as the placeholder defined in the story is {geography} instead of `{dataset}` in the query. We can't know it by using only the `sql_query` param as we might need to parse it or use a regex and is troublesome.

I preferred being explicit in the client than trying to be smart in the backend.

Reply:
Agree with what you say, but to keep the code simple you could do:
```python
is_geography = self.__class__.__name__ == 'Geography'
```

(whether there's a `sql_query` or not shouldn't matter, right?)